packages/blade/src/components/Amount/index.tsx

```diff
@@ -0,0 +1 @@
+export * from './Amount';
```

Comment:
Lets have some control over what we're exposing. We're also doing export `*` in the root index which means we won't have any control if we do * everywhere
```suggestion
export { Amount } from './Amount';
export type { AmountProps } from './Amount';
```